acceptable/__main__.py

```diff
@@ -1,21 +1,22 @@
 # Copyright 2017 Canonical Ltd.  This software is licensed under the
 # GNU Lesser General Public License version 3 (see the file LICENSE).
 import argparse
-from collections import defaultdict, OrderedDict
-from importlib import import_module
 import json
 import os
 import sys
+from collections import defaultdict, OrderedDict
+from importlib import import_module
 
+import yaml
 from jinja2 import (
     ChoiceLoader,
     Environment,
     FileSystemLoader,
     PackageLoader,
 )
-import yaml
 
 from acceptable import get_metadata, lint
+from acceptable.openapi_exporter import dump_oas
```

Comment:
`acceptable.openapi_exporter.dump_oas` sounds a bit redundant to me; specifically the "exporter" and the "oas". I'd just call it `acceptable.openapi.dump()` (or `dumps()`).

Reply:
Agreed. It's now acceptable.openapi.dump()